config/datacenters.csv

```diff
@@ -47,6 +47,10 @@
 5.199.173.240,5.199.173.247,OneProvider,http://oneprovider.com/
 5.199.174.104,5.199.174.111,OneProvider,http://oneprovider.com/
 5.252.68.0,5.252.69.255,Host1Plus UK,https://www.host1plus.com/
+5.253.204.0,5.253.204.255,M247,https://www.m247.com/
```

Comment:
These all appear consecutive - could it be combined into one line? ie, would something like

```
5.253.204.0,5.253.207.255,M247,https://www.m247.com/
```

work?

Reply:
We discussed this in the chatfilters channel: m247 typically will have 4 blocks in a row, all in different countries.  Marty said that he usually looks up all 4 and puts them in separately since they can be registered to different providers and it makes it easier to rearrange later.  If we want to start combining blocks, there are lots of entries that have already been added this way.
![image](https://user-images.githubusercontent.com/30205550/64465931-9ef4d380-d0d4-11e9-9068-354a6d6eb586.png)

Reply:
Thanks for the explanation!